applications/nrf5340_audio/src/bluetooth/ble_core.c

```diff
@@ -115,16 +116,12 @@ static int controller_leds_mapping(void)
 {
 	int ret;
 
-	ret = ble_hci_vsc_map_led_pin(PAL_LED_ID_CPU_ACTIVE,
-				      DT_GPIO_FLAGS_BY_IDX(DT_NODELABEL(rgb2_green), gpios, 0),
-				      DT_GPIO_PIN_BY_IDX(DT_NODELABEL(rgb2_green), gpios, 0));
+	ret = ble_hci_vsc_map_led_pin(PAL_LED_ID_CPU_ACTIVE, GPIO_ACTIVE_HIGH, 29);
```

Comment:
I'll comment here before someone else does. This is a special situation as we must use VSC to tell the NET core to use these pins. We can no longer use DT_NODELABEL tools to get these values, as the APP core no longer knows about this. I can define these, but then they will be multiply defined. Suggestions are welcome.